frontend/src/main/web/src/components/TableRow.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react'
+import { merge } from 'lodash'
+import { Row } from './'
+
+const classes = {
+  base: {
+    bd: 'Bdb(bd1) Bdc(light)',
+    h: 'H(r2)'
+  },
+  highlight: {
+    trs: 'Trs(aeo)',
+    hover: {
+      bgc: 'Bgc(light):h'
+    }
+  },
+  selected: {
+    bgc: 'Bgc(light)',
+    custom: 'row--selected'
+  }
+}
+
+const TableRow = ({
```

Comment:
Do we really need to mentioned its a styled `<Row>` component given the css declaration is pretty obvious on top and in the component as well, `theme={themedState}`. The chances of user going to use this component will have a little knowledge how the rendering works and able to see how the styling is being used. Unless we are expecting someone with no reactjs skills starting to use our component.